app/src/main/java/com/github/sdp/mediato/SearchBar.java

```diff
@@ -5,132 +5,129 @@
 import android.os.Bundle;
 import android.view.View;
 import android.widget.Button;
-import android.widget.ListView;
+import android.widget.GridView;
 import android.widget.SearchView;
 import android.widget.TextView;
+
 import androidx.appcompat.app.AppCompatActivity;
 
-public class SearchBar extends AppCompatActivity implements View.OnClickListener,
-    SearchView.OnQueryTextListener {
+public class SearchBar extends AppCompatActivity implements View.OnClickListener, SearchView.OnQueryTextListener {
```

Comment:
Since all of this code got moved into the SearchFragment now I think this activity can be deleted. If not, could you rename this to SearchBarActivity please so it has the same naming convention as the other activities.